examples/AN00161_using_SPI_slave/src/main.xc

```diff
@@ -127,10 +127,11 @@ void reg_file(server spi_slave_callback_if i_spi,
  */
 void app(client reg_if reg) {
     reg.set_reg(0, 0xfe);
-    debug_printf("APP: Set register 1 to 0xFE\n");
+    printstr("APP: Set register 1 to 0xFE\n");
     while (1) {
         delay_microseconds(20);
-        debug_printf("APP: Register 1 is 0x%x\n", reg.get_reg(1));
```

Comment:
Can't see why you went to the trouble of removing use of lib logging here!? Not a big issue tho.

Reply:
The library remained free of the logging dependency with the exception of the AN00161 example, and it seemed the simple prints used didn't justify the logging library. Agreed however that's not a big one since we use the logging library everywhere, and it's not a dependency that could cause any difficulty whatsoever.